oshmem/shmem/c/profile/defines.h

```diff
@@ -278,6 +367,16 @@
 #define shmem_ulong_g                pshmem_ulong_g
 #define shmem_ulonglong_g            pshmem_ulonglong_g
 #define shmem_longdouble_g           pshmem_longdouble_g
+#define shmem_int8_g                 pshmem_int8_g
+#define shmem_int16_g                pshmem_int16_g
+#define shmem_int32_g                pshmem_int32_g
+#define shmem_int64_g                pshmem_int64_g
+#define shmem_uint8_g                pshmem_uint8_g
+#define shmem_uint16_g               pshmem_uint16_g
+#define shmem_uint32_g               pshmem_uint32_g
+#define shmem_uint64_g               pshmem_uint64_g
+#define shmem_size_g                 pshmem_size_g
+#define shmem_ptrdiff_g              pshmem_ptrdiff_g
 
 #define shmemx_int16_g               pshmemx_int16_g
 #define shmemx_int32_g               pshmemx_int32_g
```

Comment:
@hoopoepg same here ..

Reply:
Didn't removed this to not break existing software where this API is used